ci/centos7-ci.sh

```diff
@@ -43,9 +42,9 @@ install_dependencies() {
 
     # install openresty to make apisix's rpm test work
     yum install -y yum-utils && yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
-    export luajit_xcflags="-DLUAJIT_ASSERT -DLUAJIT_NUMMODE=2 -DLUAJIT_ENABLE_LUA52COMPAT -O0"
-    export debug_args=--with-debug
```

Comment:
Why do you remove export debug_args=--with-debug, all debug log will be suppressed

Reply:
`./build-apisix-runtime.sh latest` contains debug information.

Without `with-debug`, our test cases cannot pass.